env/.cmdrc_sample.json

```diff
@@ -26,6 +23,7 @@
     "REACT_APP_NEW_RELIC_APPLICATION_ID": "dummy-new-relic-application-id",
     "REACT_APP_NEW_RELIC_LICENSE_KEY": "dummy-new-relic-license-key",
     "REACT_APP_NEW_RELIC_TRUST_KEY": "dummy-new-relic-trust-key",
-    "REACT_APP_USER_VOICE_KEY": "dummy-user-voice-key"
+    "REACT_APP_USER_VOICE_KEY": "dummy-user-voice-key",
+    "REACT_APP_LOBSTER_URL": "https://logkeeper.mongodb.org"
```

Comment:
This should also be added to the [setup-credentials](https://github.com/evergreen-ci/spruce/blob/5ca971fd00f3368b2cda6b65687ca606c6c224d6/cli-scripts/setup-credentials.js#L10) script